### PR TITLE
Add TCB->TDB flag for EQUAD created from TNEQUAD

### PR DESCRIPTION
**Pull request by dlakaplan**
Also add explicit printing of which parameters cause TCB->TDB conversion to fail

### CODE REVIEW

**Comment by dlakaplan** [comment]
@abhisrkckl : this is very minor

**Comment by abhisrkckl** [comment]
This looks good to me. Shall I merge this?

File: CHANGELOG-unreleased.md [hunk 1]
```diff
@@ -10,5 +10,7 @@ the released changes.
 ## Unreleased
 ### Changed
 ### Added
+* When TCB->TDB conversion info is missing, will print parameter name
 ### Fixed
+* When EQUAD is created from TNEQ, has proper TCB->TDB conversion info
```
Comment (abhisrkckl) [inline CHANGELOG-unreleased.md:15]: The bullet points are indicated with "-", not "*"

Comment (dlakaplan) [reply CHANGELOG-unreleased.md]: fixed

**Comment by dlakaplan** [comment]
> This looks good to me. Shall I merge this?

I think you can, although I didn't search exhaustively to see if any other parameters were missing the scaling info.

**Pull request merged**